lib/models/user.rb

```diff
@@ -3,38 +3,43 @@
 require_relative '../database_connection'
 require 'pry'
 
+# User class
 class User
-
   def self.create(name:, username:, email:, password:)
     encrypted_password = BCrypt::Password.create(password)
 
-    result = DatabaseConnection.query("INSERT INTO users (name, username, email, password) \
-                              VALUES('#{name}', '#{username}', '#{email}', '#{encrypted_password}') \
-                              RETURNING id, name, username, email;").first
-     build_user(result)
+    result = DatabaseConnection.query(
+      "INSERT INTO users (name, username, email, password) \
+       VALUES('#{name}', '#{username}', '#{email}', '#{encrypted_password}') \
+       RETURNING id, name, username, email;"
+    ).first
+    build_user(result)
   end
 
   def self.find(id:)
     return nil unless id
+
     result = select_by(column: 'id', value: id)
     build_user(result)
   end
 
   def self.authenticate(email:, password:)
     result = select_by(column: 'email', value: email)
-    if result && password_check(password: password, result: result)
-        build_user(result)
-    else
-      nil
-    end
+    build_user(result) if result && password_check(
+      password: password,
+      result: result
+    )
```

Comment:
Does result return nil in this case as other code depends on it returning `nil`?